packages/protocol/contracts/common/GoldToken.sol

```diff
@@ -2,17 +2,17 @@ pragma solidity ^0.5.13;
 
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 
 import "./CalledByVm.sol";
-import "./Freezable.sol";
 import "./Initializable.sol";
 import "./interfaces/ICeloToken.sol";
 import "../common/interfaces/ICeloVersionedContract.sol";
 
 contract GoldToken is
   Initializable,
   CalledByVm,
-  Freezable,
```

Comment:
If we remove the `Freezable` interface also, we need to remove the `onlyWhenNotFrozen` modifiers as well
https://app.circleci.com/pipelines/github/celo-org/celo-monorepo/33230/workflows/e3014f01-e2fa-4e5f-b5d8-b84d7a1588cf/jobs/422147/parallel-runs/0/steps/0-114?invite=true